int32[2] size
---
sensor_msgs/Image image
